resnet.py

```diff
@@ -0,0 +1,94 @@
+import json
+from pathlib import Path
+from typing import Any, Optional, Tuple
+
+import torch
+import torchvision.models as models
+from pytorch_lightning import LightningModule
+from torch import Tensor
+from torch.nn import BCEWithLogitsLoss, Linear
+from torch.optim import AdamW
+from torch.optim.lr_scheduler import CosineAnnealingLR
+from torchio.transforms import RescaleIntensity
+
+from config import Config
+
+
+class LitResnet(LightningModule):
```

Comment:
Nit: Remove reference to Lightning in name
```suggestion
class Resnet(LightningModule):
```

Reply:
Done